src/externs/datatables.js

```diff
@@ -6,3 +6,13 @@
  * @param {Object=} opt_params
  */
 jQuery.prototype.DataTable = function(opt_params) {};
+
+/**
+ * @param {Object=} opt_params
+ */
+jQuery.prototype.rows = function(opt_params) {};
```

Comment:
rows() is a function under [DataTables API instance](http://datatables.net/reference/type/DataTables.Api). It's not a function of jQuery. Please typedef DataTables API instance and then define the functions under it.